changes/v5.0.14-zh.md

```diff
@@ -1,18 +0,0 @@
-# v5.0.14
-
-## 增强
-
-- 允许环境变量重载 `host:port` 值时不使用引号 [#9614](https://github.com/emqx/emqx/pull/9614)。
-  在此修复前，环境变量中使用 `host:port` 这种配置时，用户必须使用引号，例如：
-  `env EMQX_BRIDGES__MQTT__XYZ__SERVER='"localhost:1883"'`。
-  此修复后，可以不使用引号，例如 `env EMQX_BRIDGES__MQTT__XYZ__SERVER='localhost:1883'`。
-
-- 通过 API 查询 `bridges` 信息时将混淆响应中的敏感数据 [#9593](https://github.com/emqx/emqx/pull/9593/)。
-
-- 使得规则引擎的行为与桥梁的指标行为更加一致：如果一个规则引擎被禁用，其指标现在会被重置 []()。
-
-## 修复
-
-- 修复了测试GCP PubSub可能泄露内存的问题，以及其JWT令牌第二次刷新失败的问题。 [#9640](https://github.com/emqx/emqx/pull/9640)
```

Comment:
should be 9641

changes/v5.0.14/fix-9640.zh.md --> changes/v5.0.14/fix-9641.zh.md